plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/transform/ModuleWrappingTransformation.xtend

```diff
@@ -590,42 +561,47 @@ class ModuleWrappingTransformation extends Transformation {
 			val ste = subExpr.rewiredTarget
 			if( ste.isExported ) {
 				val container = expr.eContainer;
-				if( container.isAppendableStatement ) {
+				if( container.isAppendableStatement) {
 					// case 1 : contained in simple statement, then we can issue es 2nd statement just after.
 					insertAfter(container, _ExprStmnt(_N4ExportExpr(ste,steFor_$n4Export)));
 				} else {
-					exprReplacement(expr, ste);
+					switch (expr.op) {
+						case INC: exprReplacement(expr, ste)
+						case DEC: exprReplacement(expr, ste)
+						default: {/*other unary operators do not modify original value, hence no replacement*/}
+					}
 				}
 			}
 		}
 	}
 
 
 	def private dispatch void inferExportCall(AssignmentExpression expr) {
-
 		val lhs = expr.lhs;
-		val isTopLevel_but_not_return = expr.eContainer.isAppendableStatement;
 		switch (lhs) {
 			IdentifierRef_IM: {
 				val ste = lhs.rewiredTarget;
 				if( ste.isExported ) {
-					if( isTopLevel_but_not_return ) { insertAfter(expr.eContainer, _ExprStmnt(_N4ExportExpr(ste,steFor_$n4Export))); }
+					val container = expr.eContainer
+					if( container.isAppendableStatement) {
```

Comment:
Is this yet another coding style? (I am referring to the space character on after the opening bracket, but without any space character before the closing bracket.)